### PR TITLE
Fix for #35

### PR DESCRIPTION
**Pull request by stereobooster**

### CODE REVIEW

**Comment by peterbe** [comment]
We need to be better at up-keeping the documentation. Can you please mention this in the README too?

File: package.json [hunk 1]
```diff
@@ -38,7 +38,8 @@
     "typescript": "^2.7.0-dev.20171025"
   },
   "scripts": {
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "tsc": "tsc -p ."
```
Comment (peterbe) [inline package.json]: Did this slip in from another branch?

Comment (stereobooster) [reply package.json]: nope. decided to add it. So we can `yarn tsc`

**Comment by stereobooster** [comment]
Updated

**Comment by peterbe** [comment]
Again, Thank you so much! I've updated the CHANGELOG too.

**Pull request merged**